pkgs/top-level/all-packages.nix

```diff
@@ -10440,7 +10440,10 @@ with pkgs;
 
   seexpr = callPackage ../development/compilers/seexpr { };
 
+  selectdefaultapplication = libsForQt5.callPackage ../applications/misc/selectdefaultapplication { };
+
   semgrep = python3.pkgs.callPackage ../tools/security/semgrep { };
+
```

Comment:
Those are very closely related that's why there is no new line between them.

Reply:
Removed!